clustered-indexes.md

```diff
@@ -0,0 +1,188 @@
+1 row in set (0.00 sec)
+```
+
+The `EXPLAIN` results look similar, but in the second example TiDB must first read the `PRIMARY KEY` index on `guid` in order to find the `handle` value. This is more obvious in the following example where the value of the `PRIMARY KEY` is not in the index on `does_not_cluster_by_default.b`. An extra lookup must be performed on the table rows (`└─TableFullScan_5`) to convert the `handle` to the `PRIMARY KEY` value of `guid`:
+
+```sql
+EXPLAIN SELECT id FROM always_clusters_in_all_versions WHERE b = 'aaaa';
+EXPLAIN SELECT guid FROM does_not_cluster_by_default WHERE b = 'aaaa';
+```
+
+```sql
++--------------------------+---------+-----------+---------------------------------------------------+-------------------------------------------------------+
+| id                       | estRows | task      | access object                                     | operator info                                         |
++--------------------------+---------+-----------+---------------------------------------------------+-------------------------------------------------------+
+| Projection_4             | 0.00    | root      |                                                   | test.always_clusters_in_all_versions.id               |
+| └─IndexReader_6          | 0.00    | root      |                                                   | index:IndexRangeScan_5                                |
+|   └─IndexRangeScan_5     | 0.00    | cop[tikv] | table:always_clusters_in_all_versions, index:b(b) | range:["aaaa","aaaa"], keep order:false, stats:pseudo |
++--------------------------+---------+-----------+---------------------------------------------------+-------------------------------------------------------+
+3 rows in set (0.01 sec)
+
++---------------------------+---------+-----------+-----------------------------------+------------------------------------------------+
+| id                        | estRows | task      | access object                     | operator info                                  |
++---------------------------+---------+-----------+-----------------------------------+------------------------------------------------+
+| Projection_4              | 0.00    | root      |                                   | test.does_not_cluster_by_default.guid          |
+| └─TableReader_7           | 0.00    | root      |                                   | data:Selection_6                               |
+|   └─Selection_6           | 0.00    | cop[tikv] |                                   | eq(test.does_not_cluster_by_default.b, "aaaa") |
+|     └─TableFullScan_5     | 2.00    | cop[tikv] | table:does_not_cluster_by_default | keep order:false, stats:pseudo                 |
++---------------------------+---------+-----------+-----------------------------------+------------------------------------------------+
+4 rows in set (0.00 sec)
+```
+
+# With clustered indexes
```

Comment:
The previous section also discusses clustered indexes ("TiDB has always supported clustered indexes, provided the following criteria are true:"). So this title is a bit confusing.

Reply:
Agreed. Can we change it to "Changes since in TiDB 5.0"?